blocks/library/latest-posts/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlockType } from '../../api';
+import { getLatestPosts } from './data.js';
+
+registerBlockType( 'core/latestposts', {
+	title: wp.i18n.__( 'Latest Posts' ),
```

Comment:
Minor: Should we be consistent about importing `wp.*` and use an `import { __ } from 'i18n';` here?